Wikipedia/Code/ExploreViewController.swift

```diff
@@ -320,7 +320,9 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
 
     private func setupFetchedResultsController() {
         let fetchRequest: NSFetchRequest<WMFContentGroup> = WMFContentGroup.fetchRequest()
-        fetchRequest.predicate = NSPredicate(format: "isVisible == YES && (placement == NULL || placement == %@)", "feed")
+        let now: Date = (Date() as NSDate).wmf_midnightUTCDateFromLocal
+        let oldestDate = now.addingTimeInterval(-(60 * 60 * 24 * Double(WMFExploreFeedMaximumNumberOfDays)))
```

Comment:
Certainly nothing wrong with this approach and fine as is, since we're just trying to roughly filter the displayed results, but here's an alternative if you're looking to clean up some of magic numbers and have it read a little more directly:

```suggestion
let today = NSDate().wmf_midnightUTCDateFromLocal as Date
let oldestDate = Calendar.current.date(byAdding: .day, value: -WMFExploreFeedMaximumNumberOfDays, to: today) ?? today
```

Reply:
Agreed, that does feel much better. Updated!